src/overlays/actors/ovl_Bg_Dodoago/z_bg_dodoago.c

```diff
@@ -143,9 +143,9 @@ void BgDodoago_Init(Actor* thisx, PlayState* play) {
     Collider_InitCylinder(play, &this->mainCollider);
     Collider_InitCylinder(play, &this->leftCollider);
     Collider_InitCylinder(play, &this->rightCollider);
-    Collider_SetCylinder(play, &this->mainCollider, &this->dyna.actor, &sColCylinderInitMain);
-    Collider_SetCylinder(play, &this->leftCollider, &this->dyna.actor, &sColCylinderInitLeftRight);
-    Collider_SetCylinder(play, &this->rightCollider, &this->dyna.actor, &sColCylinderInitLeftRight);
+    Collider_SetCylinder(play, &this->mainCollider, &this->dyna.actor, &sMainColliderCylinderInit);
+    Collider_SetCylinder(play, &this->leftCollider, &this->dyna.actor, &sLeftRightColliderCylinderInit);
+    Collider_SetCylinder(play, &this->rightCollider, &this->dyna.actor, &sLeftRightColliderCylinderInit);
```

Comment:
This doesnt have to happen here. But I think in general the names of "left/right/main" can change.
The "left" and "right" colliders are oc colliders that help handle the bombs as the player throws them in. It kills the speed of the bomb actor and sets a longer bomb timer so it suits the cutscene.

The "main" collider is a cylinder inside the head that detects the actual bomb explosion and lights the eyes